etna/search/views.py

```diff
@@ -294,9 +298,11 @@ def website_search(request):
                 Aggregation.CATALOGUE_SOURCE,
                 Aggregation.CLOSURE,
                 Aggregation.COLLECTION,
-                Aggregation.GROUP,
                 Aggregation.LEVEL,
                 Aggregation.TOPIC,
+                # Fetching more groups so that we receive a counts
+                # for any bucket/tab options we might be showing
+                f"{Aggregation.GROUP}:30",
             ],
```

Comment:
Can we have it as a constant like 'AGGREGATION_SIZE' used in catalogue_search_long_filter_chooser than hardcode?